exc_cell_splitting/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "region": {
+        "name": "Isocortex",
+        "query": "Isocortex",
+        "attribute": "acronym",
+        "with_descendants": true
+
+    },
+    "layers": {
+        "names": ["layer_1", "layer_2", "layer_3", "layer_4", "layer_5", "layer_6"],
+        "queries": ["@.*1[ab]?$", "@.*2[ab]?$", "@.*[^/]3[ab]?$", "@.*4[ab]?$", "@.*5[ab]?$", "@.*6[ab]?$"],
+        "attribute": "name",
+        "with_descendants": false
```

Comment:
@dkeller9 Was this in an example somewhere? I'm not sure it's supported...

Reply:
Also, it appears some of the regexs are matching things like `"Field CA2`; is that expected?

Reply:
It's stuff that Luc Guyot wrote in this very repo, e.g.: https://github.com/BlueBrain/atlas-densities/blob/eca77235fff3bbf1d3f9e8be5f1bbaac589a00b4/atlas_densities/app/data/metadata/isocortex_metadata.json

Reply:
Thanks; something seems strange with it, I will look into it.

Reply:
This regex work if you precise that you are looking for Isocortex regions (see atlas_densities.utils.create_layered_volume). 
BTW, what is the point of re-creating this metadata file since it is already in the repo?

Reply:
@drodarie how do you load the metadata from the repo and specify that you are looking for isocortex? I'd love to just use the one in the repo but could not figure out how to do that.

Reply:
From the atlas_densities.utils.create_layered_volume:

```
metadata_layers = metadata["layers"]
region_ids = region_map.find(
      metadata["region"]["query"],
      attr=metadata["region"]["attribute"],
      with_descendants=metadata["region"].get("with_descendants", False),
  ) # set of ids of Isocortex
layer_to_ids = {}
for (index, query) in enumerate(metadata_layers["queries"], 1):
        layer_to_ids[index + 1] = region_ids & region_map.find(
            query,
            attr=metadata_layers["attribute"],
            with_descendants=metadata_layers.get("with_descendants", False),
        ) # set of ids of one layer

```
The last for loop gets the ids of each layer and store them into a dictionary from layer number
So you just have to open the metadata file in your app function: 
```
with open(metadata_path, "r", encoding="utf-8") as file_:
      metadata = json.load(file_)
      assert_metadata_content(metadata)
```
From the ids, you can either get all the voxels of each layer:
```
layer_index = 1 # layer 1
layers = np.zeros_like(annotated_volume, dtype=np.uint8)
voxel_in_layer = np.isin(annotated_volume, list(layer_to_ids[layer_index]))  # convert set to list
```